### DISCUSSION TITLE
PredictionRigidbody vs Rigidbody

### DISCUSSION THREAD

**Discussion by maxgmer** [FirstGearGames/FishNet#650] [Documentation]
Would be nice to have a comparison paragraph about `PredictionRigidbody` vs `Rigidbody` here:
https://fish-networking.gitbook.io/docs/manual/guides/prediction/version-2/predictionrigidbody

Why do I need `PredictionRigidbody`? (e.g. `Reconcile()` method reconciles forces, but what if I don't set any forces and only do `rigidbody.velocity = myVelocity;`? Do I still need it?

What problems that `Rigidbody` has does `PredictionRigidbody` solve?

Maybe a noob question, but there is a lot of networking noobs just like me.

**Comment by FirstGearGames** [comment]
Thanks for your insight on the guide. I'll update it to be more clear.

**Comment by FirstGearGames** [comment]
Added to roadmap.
